aten/src/ATen/native/transformers/hip/flash_attn/ck/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ endif()
 
 execute_process(
   COMMAND python3 ${CMAKE_CURRENT_LIST_DIR}/../../../../../../../../third_party/composable_kernel/example/ck_tile/01_fmha/generate.py
```

Comment:
Nit can we update this relative path to use the root project dir, this relative path is wayyyyyy to long